src/tests/ftest/util/data_mover_utils.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from command_utils_base import FormattedParameter
+from command_utils_base import BasicParameter
+from command_utils import ExecutableCommand
+from job_manager_utils import Mpirun
+
+
+class DataMoverCommand(ExecutableCommand):
+    """Defines a object representing a datamover command."""
+
+    def __init__(self, namespace, command):
+        """Create a datamover Command object."""
+        super(DataMoverCommand, self).__init__(namespace, command)
+
+        # datamover options
+
+        # IO buffer size in bytes (default 1MB)
+        self.blocksize = FormattedParameter("--blocksize {}")
+        # DAOS source pool
+        self.daos_src_pool = FormattedParameter("--daos-src-pool {}")
+        # DAOS destination pool
+        self.daos_dst_pool = FormattedParameter("--daos-dst-pool {}")
+        # DAOS source container
+        self.daos_src_cont = FormattedParameter("--daos-src-cont {}")
+        # DAOS destination container
+        self.daos_dst_cont = FormattedParameter("--daos-dst-cont {}")
+        # DAOS service level
+        self.daos_svcl = FormattedParameter("--daos-svcl {}", 0)
+        # Source service level
+#        self.daos_src_svcl = FormattedParameter("--daos-src-svcl {}", 0)
+        # Destination service level
+#        self.daos_dst_svcl = FormattedParameter("--daos-dst-svcl {}", 0)
+        # DAOS prefix for unified namespace path
+        self.daos_prefix = FormattedParameter("--daos-prefix {}")
+        # read source list from file
+        self.input_file = FormattedParameter("--input {}")
+        # work size per task in bytes (default 1MB)
+        self.chunksize = FormattedParameter("--chunksize {}")
+        # preserve permissions, ownership, timestamps, extended attributes
+        self.preserve = FormattedParameter("--preserve", False)
+        # use synchronous read/write calls (O_DIRECT)
+        self.synchronous = FormattedParameter("--synchronous", False)
```

Comment:
```suggestion
        # open files with O_DIRECT
        self.direct= FormattedParameter("--direct", False)
```
This parameter has been renamed.

Reply:
Done